meta-refkit-extra/recipes-convnet/boost/boost_%.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGECONFIG += "python"
```

Comment:
Ah, I see. ```python``` isn't enabled by default. I got confused by what you wrote earlier: "The bbappend should not be needed anymore."